StandardCyborgFusion/Public/Mesh/SCMeshTexturing.h

```diff
@@ -28,6 +28,14 @@ typedef NS_ENUM(NSUInteger, SCMeshTexturingAPIError) {
     SCMeshTexturingAPIErrorArgument = 4 // Invalid arguments were passed in
 };
 
+/**
+ Use this to select a meshing strategy when reconstructing a textured mesh
+ */
+typedef NS_ENUM(NSUInteger, SCMeshTextureColoringStrategy) {
```

Comment:
If you use vertex colors instead of generating a UV-mapped texture, it’s not really a texture coloring strategy since no texture is generated. How about this for the name instead?

```suggestion
/**
 Use this to select a coloring strategy when generating the textured mesh
 */
typedef NS_ENUM(NSUInteger, SCMeshColoringStrategy) {
```